demos/browser/app/meeting/meeting.ts

```diff
@@ -551,11 +554,11 @@ export class DemoMeetingApp implements AudioVideoObserver, DeviceChangeObserver
   }
 
   async initializeMeetingSession(configuration: MeetingSessionConfiguration): Promise<void> {
-    let logger :MeetingSessionPOSTLogger | ConsoleLogger = null;
-    if (!(DemoMeetingApp.BASE_URL.includes('localhost') || DemoMeetingApp.BASE_URL.includes("127.0.0.1"))) {
-      logger = new MeetingSessionPOSTLogger('SDK', configuration, 85, 1150, `${DemoMeetingApp.BASE_URL}logs`, LogLevel.INFO);
-    } else {
+    let logger: Logger;
+    if (isLocal) {
       logger = new ConsoleLogger('SDK', LogLevel.DEBUG);
+    } else {
+      logger = new MeetingSessionPOSTLogger('SDK', configuration, 85, 1150, `${DemoMeetingApp.BASE_URL}logs`, LogLevel.INFO);
```

Comment:
Can add constants for 85 and 1150?

Reply:
- Added `readonly` constants.
- Config dictionary vs. individual parameters - Chime SDK has been following the latter pattern, passing individual parameters. e.g., [SignalingClientSubscribe](https://github.com/aws/amazon-chime-sdk-js/blob/master/src/signalingclient/SignalingClientSubscribe.ts#L21)